### PR TITLE
Adding a Traffic Ops client method to login with a certificate

### PR DESCRIPTION
**Pull request by srijeet0406**
This PR is not related to any issue. It adds a Traffic Ops client method to be able to login with certs.

<hr/>

## Which Traffic Control components are affected by this PR?

- Traffic Ops

## What is the best way to verify this PR?

Make sure all the tests pass.
Manual testing:
Follow the steps in https://github.com/apache/trafficcontrol/pull/7392

## If this is a bugfix, which Traffic Control versions contained the bug?

- master

## PR submission checklist
- [x] This PR has tests 
- [ ] This PR has documentation 
- [x] This PR has a CHANGELOG.md entry 
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://apache.org/security) for details)

### CODE REVIEW

File: .github/actions/to-integration-tests/cdn.json [hunk 1]
```diff
@@ -1,8 +1,15 @@
 {
+	"client_certificate_authentication" : {
+		"root_certificates_directory" : "/etc/pki/tls/traffic_ops/"
+	},
 	"disable_auto_cert_deletion": false,
 	"use_ims": true,
 	"role_based_permissions": true,
 	"traffic_ops_golang": {
+		"tls_config": {
+			"MinVersion": 769,
```
Comment (ocket8888) [inline .github/actions/to-integration-tests/cdn.json:10]: please in the name of all that is good and sacred in this world tell me that we don't seriously use hard-coded, implementation-defined, numeric constants to define security-related configuration

Comment (zrhoffman) [reply .github/actions/to-integration-tests/cdn.json:10]: This was suggested before in https://github.com/apache/trafficcontrol/pull/4573#discussion_r400346222:

> 769, 770, etc are not very intuitive -- would it be better to use a string or float that we translate into `tls.VersionTLS11` from the `tls` package in the stdlib? If 770 is basically required in order for TO to even communicate w/ Riak, should we just make 770 the default, allowing people to change it if they know what they're doing?

Comment (ocket8888) [reply .github/actions/to-integration-tests/cdn.json:10]: I really thought we took that advice back then. I'd have been pushing for it all this time if I was more aware of my surroundings

Comment (zrhoffman) [reply .github/actions/to-integration-tests/cdn.json:10]: Changing that is a little outside the scope of this PR though, so a new issue should be opened for it.

Comment (ocket8888) [reply .github/actions/to-integration-tests/cdn.json:10]: Of course. I was just hoping he was doing it wrong.

**Review by zrhoffman — APPROVED** [review]
Looks good! Just 1 comment about the cert expirations, but they can be addressed in a follow-up PR :)

**Pull request merged**